src/status_im/chat/events/console.cljs

```diff
@@ -120,8 +106,8 @@
 
 ;; TODO(janherich) remove this once send-message events are refactored
 (handlers/register-handler-fx
-  :console-respond-command
-  [(re-frame/inject-cofx :random-id-seq) re-frame/trim-v]
-  (fn [{:keys [random-id-seq]} [command]]
-    (when-let [events (console-respond-command-events command random-id-seq)]
-      {:dispatch-n events})))
+ :console-respond-command
+ [(re-frame/inject-cofx :random-id-seq) re-frame/trim-v]
+ (fn [{:keys [random-id-seq]} [command]]
+   (when-let [messages (console-respond-command-messages command random-id-seq)]
+     {:dispatch-n [[:chat-received-message/add messages]]})))
```

Comment:
Why do we `:dispatch-n` something which is vector with single item, why not `{:dispatch [:chat-received-message/add messages]}` ?